assets/sass/components/global/_googlesitekit-error-text.scss

```diff
@@ -19,6 +19,12 @@
 .googlesitekit-error-text {
 	color: $c-error;
 	font-weight: 500;
+	margin: 1rem 0;
+
+	p {
+		display: inline;
+		margin-right: 1em;
```

Comment:
@sashadoes There should be no margin set to the `p` tag, else the spacing would look weird and not natural.